website/source/docs/install/bootstrapping.html.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "docs"
+page_title: "Bootstrapping a Datacenter"
+sidebar_current: "docs-install-bootstrapping"
+description: |-
+  An agent can run in both client and server mode. Server nodes are responsible for running the consensus protocol and storing the cluster state. Before a Consul cluster can begin to service requests, a server node must be elected leader. Thus, the first nodes that are started are generally the server nodes. Bootstrapping is the process of joining these server nodes into a cluster.
+---
+
+# Bootstrapping a Datacenter
+
+An agent can run in either client or server mode. Server nodes are responsible for running the
```

Comment:
Since the doc doesn't have the context of the other guides, it might make sense to give one sentence of intro about what consul considers a datacenter and what agents are. Usually, I wouldn't recommend doing that for any doc, but this could be the first place people go to learn about consul, so I think here it makes sense.

Reply:
The definition of datacenter is in the second paragraph so I think for now I'll leave the beginning as is. Instead I linked out to the architecture doc so they an learn more if they need more context.

Reply:
For clarity, I linked out for the architecture docs.